e2e/test/provisioning/ConnectionStringParser.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace Microsoft.Azure.Devices.E2ETests.provisioning
```

Comment:
Casing on the namespace.

```suggestion
namespace Microsoft.Azure.Devices.E2ETests.Provisioning
```